README.md

```diff
@@ -45,12 +45,12 @@ full-stack type-safety.
 To get started, head over to the [docs](https://connect.build/docs/web/getting-started)
 for a tutorial. You will also find API documentation and best practices there.
 For using Connect with your favorite frontend framework, take a look at
-[connect-web-integration](https://github.com/bufbuild/connect-web-integration).
+[connect-es-integration](https://github.com/bufbuild/connect-es-integration).
 
 ## Ecosystem
 
-* [connect-web-integration](https://github.com/bufbuild/connect-web-integration):
-  Example projects using Connect-Web with various JS frameworks and tooling
+* [connect-es-integration](https://github.com/bufbuild/connect-es-integration):
+  Illustrates the integration of Connect in various JS frameworks and tooling
```

Comment:
I might be overthinking, but don't new users just want some examples to get started?  
"Integration" sounds like some work is coming up.

WDYT?

Reply:
Yep.  Updated to use the tagline for the repo.